Core/Core/Profile/ProfilePresenter.swift

```diff
@@ -0,0 +1,167 @@
+                cells.append(ProfileViewCell("lti.\(tool.domain ?? "").\(tool.definitionID)", name: tool.title) { [weak self] _ in
+                    self?.view?.launchLTI(url: tool.url)
+                })
+            }
+        }
+
+        if enrollment == .student {
+            let showGrades = env.userDefaults?.showGradesOnDashboard == true
+            cells.append(ProfileViewCell("showGrades", type: .toggle(showGrades), name: NSLocalizedString("Show Grades", comment: "")) { [weak self] cell in
+                let showGrades = (cell.accessoryView as? UISwitch)?.isOn == true
+                self?.env.userDefaults?.showGradesOnDashboard = showGrades
+                NotificationCenter.default.post(name: NSNotification.Name("redux-action"), object: nil, userInfo: [
+                    "type": "userInfo.updateShowGradesOnDashboard",
+                    "payload": [ "showsGradesOnCourseCards": showGrades ],
+                ])
+            })
+        }
+
+        if enrollment == .student || enrollment == .teacher {
+            let colorOverlay = settings.first?.hideDashcardColorOverlays != true
+            cells.append(ProfileViewCell("colorOverlay", type: .toggle(colorOverlay), name: NSLocalizedString("Color Overlay", comment: "")) { cell in
+                let colorOverlay = (cell.accessoryView as? UISwitch)?.isOn == true
+                NotificationCenter.default.post(name: NSNotification.Name("redux-action"), object: nil, userInfo: [
+                    "type": "userInfo.updateUserSettings",
+                    "pending": true,
+                    "payload": [ "hideOverlay": !colorOverlay ],
+                ])
+                UpdateUserSettings(hide_dashcard_color_overlays: !colorOverlay).fetch { (settings, _, _) in
+                    guard settings == nil else { return }
+                    NotificationCenter.default.post(name: NSNotification.Name("redux-action"), object: nil, userInfo: [
+                        "type": "userInfo.updateUserSettings",
+                        "error": true,
+                        "payload": [ "hideOverlay": !colorOverlay ],
+                    ])
+                }
+            })
+        }
+
+        if let root = helpLinks.first, helpLinks.count > 1 {
+            cells.append(ProfileViewCell("help", name: root.text) { [weak self] cell in
+                self?.view?.showHelpMenu(from: cell)
+            })
+        }
+        if enrollment == .student {
+            cells.append(ProfileViewCell("settings", name: NSLocalizedString("Settings", comment: "")) { [weak self] _ in
+                self?.view?.route(to: .profileSettings, options: [.modal, .embedInNav])
+            })
+        } else if enrollment == .teacher {
+            cells.append(ProfileViewCell("settings", name: NSLocalizedString("Settings", comment: "")) { [weak self] cell in
+                self?.view?.showTeacherSettingsMenu(from: cell)
+            })
+        }
+        if canActAsUser {
```

Comment:
I purposely allowed this while already acting as a user. This allows you to immediately act as a different user instead of changeUser -> me -> profile -> act as user.

Reply:
Ah, cool with me.